dask/config.py

```diff
@@ -434,4 +436,40 @@ def update_defaults(new, config=config, defaults=defaults):
     update(config, new, priority='old')
 
 
+def expand_environment_variables(config):
+    ''' Utility to expand environment variables in a config dictionary
+
+    This function will recursively search through any nested dictionaries
+    and/or lists.
+
+    Parameters
+    ----------
+    config : dict, iterable, or str
+        Input object to search for environment variables
+
+    Returns
+    -------
+    config : same type as input
+
+    Examples
+    --------
+    >>> expand_environment_variables({'x': [1, 2, '$USER']})
+    {'x': [1, 2, 'my-username']}
```

Comment:
does anyone know how to force-skip this example in the doc tests?

Reply:
Append `# doctest: +SKIP` t the end of the line that starts with `>>>`